apps/test/unit/p5lab/redux/animationListTest.js

```diff
@@ -227,6 +227,27 @@ describe('animationList', function() {
     return {orderedKeys: orderedKeys, propsByKey: propsByKey};
   };
 
+  let createAnimationListToMigrate = function(count) {
+    let orderedKeys = [];
+    let propsByKey = {};
+    let baseKey = 'animation';
+    for (let i = 1; i <= count; i++) {
+      let key = baseKey + '_' + i;
+      orderedKeys.push(key);
+
+      propsByKey[key] = {
+        name: key,
+        sourceUrl: 'source/v3/url',
```

Comment:
from what i can tell, this is the only line that differs from `createAnimationList` -- can we reuse that function here and add `sourceUrl` as an optional parameter?